go/runtime/bundle/manager.go

```diff
@@ -234,6 +305,8 @@ func (m *Manager) Add(path string) error {
 		return fmt.Errorf("not allowed to add key manager bundle")
 	}
 
+	manifest.Labels = labels
```

Comment:
It is obvious that this doesn't belong here. 
If you create a `Rewrite` function, that you can rewrite the manifest in `explodeBundle` and set labels there. You could also use `Rewriter` to plug it into `Rewrite`, which has two methods, one that rewrites the manifest and one that returns labels.